features/180atp/apibodyexamples/changeschedule.json

```diff
@@ -0,0 +1,32 @@
+{
+    "id": "id-bike-0001",
+    "organizationId": "usmf",
+    "productId": "Bike",
+    "quantitiesByDate": {
+        "2023-09-08": {
+            "iv": {
+                "ordered": 10
+            }
+        },
+        "2023-10-01": {
+            "pos": {
+                "inbound": 20
+            }
+        }
+    },
+    "dimensions": {
+        "siteId": "1",
+        "locationId": "11",
+        "colorId": "red",
+        "sizeId": "small"
+    }
+}
+
+
+
+{
+    "id": "ohcs~id-bike-0001",
+    "processingStatus": "success",
```

Comment:
.. so that you can have more straightforward file names, e.g., for these two bodies you may have changeschedule_request.json, changeschedule_response_ok.json, and similarly for the files below.